README.md

```diff
@@ -37,63 +32,237 @@ Let's look at each of the components in more detail.
 ### Schema
 
 A model of a schema with helpful methods for working with the schema and supported data. Schema instances can be initialized with a schema source as a url to a JSON file or a JSON object. 
-The schema is initially validated (see [validate](#validate) below), and will raise an exception if not a valid JSON Table Schema.
+The schema is initially validated (see [validate](#validate) below), and will raise an exception if not a valid JSON Table Schema.  
 
-Model requires Promise to work properly, and need to be sure that Promise available globally. You are free to choose any Promise polyfill  
+```javascript
+var Schema = require('jsontableschema').schema;
```

Comment:
tests are running in front of es6, because they are written in ES6, but the src is compatible with older node versions